src/styles/button.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-size: 14px;
+  border: none;
+  color: $color-white;
+  background-color: $color-button;
+  font-family: $font-roboto;
+  font-weight: $font-bold;
+  line-height: 16px;
+  margin: 0 16px 16px;
```

Comment:
The margin property is used here to create spacing around the button. It's recommended to handle the spacing of components outside of the component itself. Consider removing the margin from the button styles and controlling it from a parent element or a utility class.